mobilecoind/src/processed_block_store.rs

```diff
@@ -130,27 +130,25 @@ impl ProcessedTxOut {
 /// the processed blocks database.
 #[derive(Clone)]
 pub struct ProcessedBlockStore {
-    /// LMDB Environment.
+    #[cfg(test)]
+    /// The LMDB environment is only actually used in tests
```

Comment:
This is a bit funky -- the thing is, if the arc environment is dropped I think it may be unsafe to keep using the database?

I'm also sort of leery of having `cfg(test)` on field members like this because if there were a problem with dropping the environment there's no way we can catch it in unit tests, since the actual code is different when testing

If the issue is an unused warning, maybe we could do like `cfg(not test, allow_unused)`? My 2 cents

Reply:
I switched to making the test setup return the `env`, and removed the field from this struct.